### PR TITLE
Added Jetty Prometheus Dashboard

### PR DESCRIPTION
**Pull request by algchoo**
* added jetty prometheus dashboard json
* included screenshot
* updated readme and metadata

### CODE REVIEW

**Review by yqlu — APPROVED** [review]

File: dashboards/jetty/README.md [hunk 1]
```diff
@@ -9,3 +9,8 @@
 |:------------------|
 |Filename: [jetty-gce-overview.json](jetty-gce-overview.json)|
 |This dashboard has charts displaying `Sessions`, `Session Time`, `Select Calls`, `Max Session Time`, `Threads`, and `Thread Queue` from Jetty as well as charts of infrastructure related metrics for the running Jetty VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the Jetty logs in Cloud Logging.|
+
+|Jetty Prometheus Overview|
+|:------------------|
+|Filename: [jetty-prometheus-overview.json](jetty-prometheus-overview.json)|
+|This dashboard is based on prometheus metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Sessions`, `Session Time`, `Select Calls`, `Max Session Time`, `Threads`, and `Thread Queue`.|
```
Comment (yqlu) [inline dashboards/jetty/README.md:16]: Update the exporter URL -- shouldn't be jmx_exporter?

File: dashboards/jetty/metadata.yaml [hunk 2]
```diff
@@ -8,4 +8,13 @@ sample_dashboards:
...
       - id: jetty
-        platform: GCE
\ No newline at end of file
+        platform: GCE
+  -
+    category: Jetty
+    id: jetty-prometheus-overview
+    display_name: Jetty Prometheus Overview
+    description: |-
+      This dashboard is based on prometheus metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Sessions, Session Time, Select Calls, Max Session Time, Threads, and Thread Queue.
```
Comment (yqlu) [inline dashboards/jetty/metadata.yaml:17]: Same here

**Review by yqlu — CHANGES_REQUESTED** [review]
Just noticed one detail

File: dashboards/jetty/README.md (hunk 1 above)
Comment (algchoo) [reply dashboards/jetty/README.md:16]: That's the exporter that was used to retrieve metrics.

Comment (yqlu) [reply dashboards/jetty/README.md:16]: Ah my bad -- I was expecting the URL to be a specialized Jetty exporter and this was a copy-paste error, but I took a closer look at https://github.com/prometheus/jmx_exporter and figured out what was going on

**Review by yqlu — APPROVED** [review]

**Pull request merged**